ftests/016-cgget-invalid_options.py

```diff
@@ -26,7 +26,7 @@ def prereqs(config):
     # This causes issues with the error handling of this test
     if not config.args.container:
         result = consts.TEST_SKIPPED
-        cause = 'This test cannot be run outside of a container'
```

Comment:
Good catch.  Since we're using this string a lot, we should make it a constant in `consts.py`.  But that can obviously be a separate cleanup.